pkg/kube/kclient/client_test.go

```diff
@@ -288,6 +289,7 @@ func TestErrorHandler(t *testing.T) {
 func TestToOpts(t *testing.T) {
 	test.SetForTest(t, &features.InformerWatchNamespace, "istio-system")
 	c := kube.NewFakeClient()
+	c.RunAndWait(test.NewStop(t))
```

Comment:
The reason for this change is the same as [above](https://github.com/istio/istio/pull/50713/files#r1581734612).